L1Trigger/Phase2L1ParticleFlow/python/l1ctLayer1_cff.py

```diff
@@ -525,5 +580,9 @@
      l1tLayer1HGCalNoTK,
      l1tLayer1HF,
      l1tLayer1,
-     l1tLayer1EG
+     l1tLayer1Extended,
```

Comment:
@cerminar The failures in all the logs mention that this is not defined. Is this reliant on some Phase 2 development that is in a PR to the l1t-offline-branch, or one that is in a PR here to main CMSSW?

Reply:
@gpetruc Okay. That PR has been outstanding for far too long and things are beginning to pile-up behind it. I will try to get in contact with upgrade to see if they have a review or are willing to sign at this stage.